### ISSUE TITLE
Bug report  -  qqX specific can't download distros

### ISSUE THREAD

**Issue by ctamietto** [TuxVinyards/qqX#7]
when i run qqx and try to dowload a distros i get this error 

![immagine](https://github.com/TuxVinyards/qqX/assets/3806889/4b677efa-4ee6-4c6b-93cc-b3e44f4184c7)

i have installed quickemu cloning the repository and adding  the directory to the path 

![immagine](https://github.com/TuxVinyards/qqX/assets/3806889/d980a8af-6aed-457b-b9d5-f6908499ba3b)

i've used quickemu ( and quickget )  from the command line and it is working

in the same i have installed qqx cloning the repository end running ./qqX_setup_and_install

when i start qqx i get this 

![immagine](https://github.com/TuxVinyards/qqX/assets/3806889/f84d773e-8df8-45d1-ad50-e457651081cf)

i'm on fedora 39

![immagine](https://github.com/TuxVinyards/qqX/assets/3806889/f685beb9-b7c8-45ba-9996-a5329e9355de)

what can i check ?

**Comment by TuxVinyards** [comment]
Thank you for posting.   I was hoping to get things ready before anyone problems ....

Have a look in the new branch that I have just posted.  1.8.02  

I have just this minute finished testing it in Fedora 40.   I think everything is sorted.   I have also tested it with Ubuntu and Arch.

It's been a bit of a race and juggling act between helping with the new Quickemu release, which has some really good new features, and managing to get qqX stable with all the new API and other changes.

https://github.com/TuxVinyards/qqX/tree/1.8.02

There's notes in the setting file that explain more.   ....  qqX now has builtins 

Let me know how you get on.  I probably release ready.

Quickemu itself still has some problems with Windows and language support.  I have had to disable some functions but it generally works for international English.   See https://github.com/quickemu-project/quickemu/issues/1113
